ogzaf/demographics.py

```diff
@@ -27,6 +27,9 @@
 from ogcore import parameter_plots as pp
 from pandas_datareader import wb
 import matplotlib.pyplot as plt
+import requests
+from ogzaf import utils
```

Comment:
I needed to change this line to so that the `get_legacy_session` below was accessible without the `utils.` prefix:

```
from ogzaf.utils import get_legacy_session
```